test/test_nestedtensor.py

```diff
@@ -2667,6 +2669,94 @@ def grad_test_func(a, b, c):
         data = (a, b, c)
         assert gradcheck(grad_test_func, inputs=data, check_batched_grad=False)
 
+# We can probably parametrizing existing tests instead of having a separate
+# test class as we begin to support more ops. Also maybe rewrite with OpInfos.
+class TestNestedTensorSubclass(TestCase):
```

Comment:
nit: perhaps parametrize over device / dtype for these tests?

`instantiate_device_type_tests(TestNestedTensorSubclass, globals())`